ui/components/component-library/banner-tip/banner-tip.test.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable jest/require-top-level-describe */
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { renderWithUserEvent } from '../../../../test/lib/render-helpers';
+
+import { BannerTip, BANNER_TIP_LOGOS } from '.';
+
+describe('BannerTip', () => {
+  it('should render BannerTip element correctly', () => {
+    const { getByTestId, container } = render(
+      <BannerTip data-testid="bannerTip" title="BannerTip test">
+        should render BannerTip element correctly
+      </BannerTip>,
+    );
+    expect(getByTestId('bannerTip')).toHaveClass('mm-banner-tip');
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render with different logo types', () => {
+    const { getByTestId } = render(
+      <>
+        <BannerTip
+          logoType={BANNER_TIP_LOGOS.GREETING}
+          logoProps={{ 'data-testid': 'banner-tip-greeting' }}
+        >
+          should render BannerTip element correctly
+        </BannerTip>
+        <BannerTip
+          logoType={BANNER_TIP_LOGOS.CHAT}
+          logoProps={{ 'data-testid': 'banner-tip-chat' }}
+        >
+          should render BannerTip element correctly
+        </BannerTip>
+      </>,
+    );
+    expect(getByTestId('banner-tip-greeting')).toHaveClass(
```

Comment:
nit: Could also check for `img` tags here to ensure the `as` tag doesn't accidentally get changed